onnxruntime/core/optimizer/not_where_fusion.cc

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/optimizer/not_where_fusion.h"
```

Comment:
nit: maybe move this line below to be together with other includes from same dir

Reply:
this follows Google's cpp styleguide https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes